TERMS.md

```diff
@@ -608,6 +618,10 @@ Avoid using as a verb to refer to an action that precipitates a subsequent actio
 
 A storage tier that you can use to store and analyze your data with Elasticsearch and Kibana that is optimized for performance. To learn more about the service, see the introductory [blog post](https://aws.amazon.com/about-aws/whats-new/2020/05/aws-announces-amazon-elasticsearch-service-ultrawarm-general-availability/).
 
+**upper left, upper right**
```

Comment:
If we usually use upper left or upper right as adjectives, should we hyphenate upper right and upper left on this line?